styles.css

```diff
@@ -324,3 +353,128 @@ h2, h3 {
         font-size: 0.4rem;
     }
 }
+
+/* ------------------------------------ About Page --------------------------------------------- */
+
+#about{
+    background-color: var(--secondary-color);
+    padding: 1rem;
+ }
+
+ .about-container{
+    margin-left: auto;
+    display: grid;
+    justify-content: center;
+}
+
+ .about-title{
+    font-size: 2rem;
+    color: var(--primary-color);
+    text-align: center;
+ }
+
+ .about-text-styling{
+    font-size: 1rem;
+    text-align: center;
+    padding: 2rem;
+    color: var(--primary-color);
+ }
+
+ .style-image{
+    width: 100%;
+    height: auto;
+    /* padding: 1.5rem; */
+}
+
+.image-description{
+    text-align:center;
+    color: var(--primary-color);
+}
+
+/* ------------- Media Queries for about Page --------------------------- */
```

Comment:
Hmmm wouldn't it be better to put all the media queries for all the pages at the end?

Reply:
It would be a lot neater and easier to read, i'll adjust this now, thats a very good point ⭐